frontend/src/metabase/home/components/EmbedMinimalHomepage/util.ts

```diff
@@ -0,0 +1,23 @@
+export const showEmbedHomepage = () => {
+  try {
+    localStorage.setItem("showEmbedHomepage", "true");
+  } catch (e) {
+    console.error(e);
```

Comment:
localStorage could fail (if disabled, or if in incognito in some browsers).
I still want to see the error if it happens but I don't want to trigger ErrorBoundaries